classes/SpecificPrice.php

```diff
@@ -750,13 +750,7 @@ public function duplicate($id_product = false)
      */
     public static function isFeatureActive()
     {
-        static $feature_active = null;
-
-        if ($feature_active === null) {
-            $feature_active = Configuration::get('PS_SPECIFIC_PRICE_FEATURE_ACTIVE');
-        }
-
-        return $feature_active;
+        return Configuration::get('PS_SPECIFIC_PRICE_FEATURE_ACTIVE');
```

Comment:
```suggestion
        return (bool) Configuration::get('PS_SPECIFIC_PRICE_FEATURE_ACTIVE');
```

Reply:
I love removing this kind of static cache :heart:

Reply:
and it introduiced a bug ^^